tfe/resource_tfe_workspace_test.go

```diff
@@ -1564,3 +1598,18 @@ resource "tfe_workspace" "foobar_two" {
   organization       = tfe_organization.foobar.id
 }`, rInt)
 }
+
+func testAccTFEWorkspace_updateStructuredRunOutput(rInt int) string {
```

Comment:
minor nitpick on naming. What do you think about changing this to `testAccTFEWorkspace_structuredRunOutputDisabled`?

Reply:
Works for me!